src/app/app.component.ts

```diff
@@ -17,14 +16,15 @@ export class AppComponent {
     selected: string;
 
     constructor(
-        private router: Router,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        private readonly router: Router,
```

Comment:
this appears to be following the naming convention rule, so why ignore?

Reply:
With the new linting rules you have to use an underscore for private dependencies. It's fixed now

Reply:
hmm, should we follow the rule and use an underscore then?